### PR TITLE
Silva datasets

### PR DESCRIPTION
**Pull request by gallardoalba**
SILVA.yaml added. It includes the SILVA datasets

### CODE REVIEW

File: servers.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
     "url": "https://usegalaxy.eu",
     "libs": {
       "GTN.yaml": "7d84c2ac21245dad",
-      "EI.yaml": "cef78de78c2ec365"
+	"EI.yaml": "cef78de78c2ec365",
+	"SILVA.yaml": "F50c98c8a05e8bc97"
```
Comment (hexylena) [inline servers.json]:
```suggestion
      "EI.yaml": "cef78de78c2ec365",
      "SILVA.yaml": "F50c98c8a05e8bc97"
```

Comment (gallardoalba) [reply servers.json]: The SILVA ID has been modified because it was wrong. The correct one is 40160950115d555e.

Comment (hexylena) [reply servers.json]: my suggestion is w/r/t the spacing, not the ID. It seems it was made on an older commit somehow.

Comment (gallardoalba) [reply servers.json]: Yes, thanks!

**Comment by bgruening** [comment]
Perfect! Thanks @gallardoalba

**Pull request merged**